e2e/generate-test.mjs

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable no-console, import/no-extraneous-dependencies */
```

Comment:
If you haven't already, can you `chmod +x e2e/generate-test.mjs` also, please?
```suggestion
#!/usr/bin/env node
/* eslint-disable no-console, import/no-extraneous-dependencies */
```

Reply:
Can you explain why we need that?